examples/assigning_to_self_example.py

```diff
@@ -1,10 +1,10 @@
 class Assigning():
```

Comment:
@ostrokach One other stylistic matter - please omit the `()` for class definitions.